### ISSUE TITLE
How can I run Cityscapes example on a test set?

### ISSUE THREAD

**Issue by mikhailkonyk** [shrubb/box-convolutions#24]
Hello, collegues! I've trained BoxERFNet, and now I wanna run this model on a test set to evaluate it.
I checked the source code(train.py) and established 'test' in place of 'test' in 80th string. But there was falure, the evaluated metrics were incorrect(e.g. 0.0 and 0.0). Can you explain me, what I need to do to evaluate model on a test set?
I guess that problem is on 'validate' function(241th string), because confusion_matrix_update(268th string) tensors are really different in test and val sets.

**Comment by shrubb** [comment]
There is no ground truth data for the test set. You can only view your test score by signing up to Cityscapes' website and uploading your predictions there.

**Comment by mikhailkonyk** [comment]
i.e. there is no ability to run model on test set?(on your example code)

**Comment by shrubb** [comment]
Yes, if you want that, I think you'll have to change my code a bit (it should be trivial).
